### ISSUE TITLE
Some unlicensed games are not working

### ISSUE THREAD

**Issue by ghost** [0ldsk00l/nestopia#227]
The following games do not work as expected:

Chuugoku Taitei (Asia) (Unl) - Cpu Jam!
Gaiapolis (Asia) (Unl) - Cpu Jam!
Galactic Crusader (Asia) (Unl)
Hidden Chinese Chess - An Qi (Asia) (NTSC) (Unl)
Jurassic Boy (Asia) (Unl) - Cpu Jam!
Mahjong World, The - Ma Que Shi Jie (Asia) (NTSC) (Unl)
Mahjong World, The - Ma Que Shi Jie (Asia) (PAL) (Unl)
Metal Fighter (Asia) (Sachen) (Unl)
Poker Mahjong - Pu Ke Mao Que (Asia) (Unl)
Popo Team (Asia) (Unl)
Pyramid II (Asia) (PAL) (Unl) - Black Screen
Rockball (Asia) (Unl) - Black Screen
Sidewinder (Asia) (PAL) (Unl)
Silent Assault (Asia) (PAL) (Unl)
Silver Eagle (Asia) (PAL) (Unl) - Black Screen
Street Heroes (Asia) (Unl) - Cpu Jam!
Super Cartridge Ver 2 - 10 in 1 (Asia) (Unl)
Super Cartridge Ver 3 - 8 in 1 (Asia) (Unl)
Super Cartridge Ver 9 - 3 in 1 (Asia) (Unl)
Super Pang II (Asia) (Unl)
Taiwan Mahjong - Tai Wan Ma Que 16 (Asia) (Unl)
Sidewinder (Australia) (Unl)
Fan Kong Jing Ying (China) (Unl) - Black Screen
Xing Ji Zheng Ba (China) (Unl) - Black Screen
AV Super Real Pachinko (Japan) (Unl)
Mahjong Samit - Kabukicho Hen (Japan) (Unl)
Dooly Bravo Land (Korea) (Unl)
Korean Igo (Korea) (Unl) - Cpu Jam!
Magic Kid Googoo (Korea) (Unl)
BB Car (Unknown) (Unl)
Hard Drivin' (USA) (Proto) (Unl)
Puzzle (USA) (Beta) (Unl)
Racermate Challenge II (USA) (v9.03.128) (Unl) - All versions
Secret Scout in the Temple of Demise (USA) (Beta) (Unl)
Contra (Japan) (Sample)
Crisis Force (Japan) (Sample)
Elite (Unknown) (NTSC Demo)
TwinBee 3 - Poko Poko Daimaou (Japan) (Sample)
Block Out (USA) (Proto)
Free Fall (USA) (Proto)
Bugs Bunny Birthday Bash (USA) (Beta) [b] - Cpu Jam!

**Comment by carmiker** [comment]
Bugs Bunny Birthday Bash will likely never work. Racermate Challenge II is its own issue entirely, it needs the mapper support and support for the exercise bike attachment.

Every other game on this list now works in the upstream build.

**Comment by TroudhuK** [comment]
Yes but we have to spend HOURS to understand how to get this build (e.g. Nestopia 1.54.0) because binaries are not available. Thank you for the news anyway, I just successfully build a nestopia.dll instead of an exe like before, ok there is a front-end project, ok it does an .exe but alone and we have to build all the folder with dependencies and config files... I forgive, and will keep 1.53.2, available ready-to-use here.

**Comment by carmiker** [comment]
@TroudhuK congratulations on managing to build the upstream core and reference frontend. You may wish to try the new frontend if you want a preview of what is coming: https://gitlab.com/jgemu/qtea
